Source/FieldSolver/FiniteDifferenceSolver/EvolveM.cpp

```diff
@@ -181,19 +154,6 @@ void FiniteDifferenceSolver::EvolveM (
                 + dt * cons2 * ( Mz(i, j, k, 0) * ( Mz(i, j, k, 2) * Bx_ztemp - Mz(i, j, k, 0) * Bz(i, j, k))
                 - Mz(i, j, k, 1) * ( Mz(i, j, k, 1) * Bz(i, j, k) - My(i, j, k, 2) * By_ztemp));
 
-              // temporary normalized magnitude of Mz field at the fixed point
```

Comment:
Keep the normalization in EvolveM.cpp as well in case we need it in the future

Reply:
How about doing it next PR until the normalization part is validated?